### PR TITLE
Added first speaker

### PR DESCRIPTION
**Pull request by tolomaus**

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Hi! For talks, "type=ignite" isn't currently supported. The only type is "talk". You can indicate in the description and program that it's an ignite.

File: content/events/2016-ghent/program/frederic-descamps.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
++++
+
+<div class="span-15  ">
+  <div class="span-15  last ">
+  <p><strong>Title:</strong>
+Deploy HA MySQL environment using Puppet and MySQL Group Replication
+</p>
+
+<p><strong>Description:</strong></p>
+This talk is covering the automation of MySQL HA environment setup using Puppet. All members of the InnoDB Cluster will be replication from each other (no asyncrhonous master/slave replication anymore) using the new MySQL Group Replication. This replication is much easier to deploy than usual MySQL replication as every members are equals. We will use a dedicated puppet module and hiera to setup our environment and important decision and variables will be highlighted and explained. The session finishes with a full cluster deployment on Vagrant in 2 minutes.
```
Comment (bridgetkromhout) [inline content/events/2016-ghent/program/frederic-descamps.md]: Are you sure this is the description of an Ignite talk? (It doesn't sound like something that would be covered in five minutes, so, just checkin.)

File: content/events/2016-ghent/program/gil-zellner.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+</p>
+
+<p><strong>Description:</strong></p>
+
+<p>
+When my wife told me she was pregnant I was scared of the new responsibility, and was certain I would screw it up. This is how I dealt with it by reframing the problem. I shall draw comparisons between caring for a newborn baby and a production system, including analytics, hacks etc.
+</p>
+
+
+  <p><strong>Title:</strong>
```
Comment (bridgetkromhout) [inline content/events/2016-ghent/program/gil-zellner.md]: Did you intend to list two talks under this speaker? Thanks for checking!

File: content/events/2016-ghent/program/frederic-descamps.md (hunk 1 above)
Comment (tolomaus) [inline content/events/2016-ghent/program/frederic-descamps.md]: Hi Bridget thanks for checking :-)

Yes I confirm it's an ignite, I believe it was converted from a regular talk but the description has stayed the same

File: content/events/2016-ghent/program/gil-zellner.md (hunk 2 above)
Comment (tolomaus) [inline content/events/2016-ghent/program/gil-zellner.md]: Yes indeed, this person will do two ignites.

**Comment by tolomaus** [comment]
""type=ignite" isn't currently supported"

I see. This PR is only just for getting the speakers page ready. So shall I send you a corrected PR right now or can I postpone it when we create the program page?

Thanks

**Comment by bridgetkromhout** [comment]
I will merge this, but the speaker info won't show on the individual talk pages until you change them to type=talk.

**Pull request merged**